source/topic1.rst

```diff
@@ -128,73 +131,78 @@ Is this class easy?
    * ...
    * Who here knows how to build a house?
 
-Each of the things you will learn are somewhat simple on their own, but using them together to achieve something is challenging.   
+
+* Each of the things you will learn are perhaps simple on their own, but using them together to achieve something is challenging.   
    
-These topics really are going to be shockingly different from what you are probably used to. Very few classes can prepare you for the way of thinking that will be required for this class.
+* These topics really are going to be shockingly different from what you are probably used to. Very few classes can prepare you for the way of thinking that will be required for this class.
 
-Unfortunately, intro to CS classes do often have an unusually high fail rate.
+* Unfortunately, intro to CS classes do often have an unusually high fail rate.
 
 
 So I should be terrified?
-^^^^^^^^^^^^^^^^^^^^^^^^^
+=========================
 
-No. Really really no. It's actually not that bad, but you should be prepared to be challenged (but that's why we're all here, right?). 
+* No. 
+* It's actually not that bad, but you should be prepared to be challenged 
```

Comment:
I'm not sure I would word it like that. Sounds like dunking on those that do end up failing. Perhaps adding something like this is sufficient to set the tone:
> It's actually not that bad _if you put in the effort_